.github/ISSUE_TEMPLATE/request_cluster_resources.md

```diff
@@ -0,0 +1,31 @@
+---
+name: General cluster-scoped resource request
+about: "Requesting resources which require cluster-admin: CRDs, ClusterRoles, Operators, Quotas"
+title: ""
+labels: onboarding
+assignees: ""
+---
+
+### Requested actions
+
+#### Operator request
+
+If you want to request an operator to be deployed either with all namespaces visibility or with a single namespace visibility, please state here which operator you are interested in. Please point us to the any documentation to this operator directly.
```

Comment:
`Please point us to the any documentation to this operator directly.` -> `Please point us to any documentation about this operator directly`

Reply:
applied